service/common/util/ConnectionUri.kt

```diff
@@ -6,23 +6,21 @@
 
 package com.typedb.studio.service.common.util
 
-import java.net.URI
-import java.net.URISyntaxException
 import java.net.URLDecoder
 import java.nio.charset.Charset
 
 object ConnectionUri {
-    private const val SCHEME_CLOUD = "typedb-cloud"
-    private const val SCHEME_CORE = "typedb-core"
+    private const val SCHEME_CLOUD = "typedb-cloud://"
+    private const val SCHEME_CORE = "typedb-core://"
```

Comment:
Now that we aren't using the built-in `URI` class, we care about the presence of the `://`